src/Volume.ts

```diff
@@ -1,96 +1,73 @@
-import { Vector3 } from "three";
+import { Vector2, Vector3 } from "three";
 
 import Channel from "./Channel";
 import Histogram from "./Histogram";
 import { getColorByChannelIndex } from "./constants/colors";
 import { LoadSpec } from "./loaders/IVolumeLoader";
 
-/* eslint-disable @typescript-eslint/naming-convention */
-export interface ImageInfo {
+export type ImageInfo = {
   name: string;
-  version: string;
-
-  /** X size of the *original* (not downsampled) volume, in pixels */
-  width: number;
-  /** Y size of the *original* (not downsampled) volume, in pixels */
-  height: number;
-  /** Number of rows of z-slice tiles in the texture atlas */
-  rows: number;
-  /** Number of columns of z-slice tiles in the texture atlas */
-  cols: number;
-  /** Width of a single atlas tile in pixels */
-  tile_width: number;
-  /** Height of a single atlas tile in pixels */
-  tile_height: number;
-  /** Number of tiles in the texture atlas (or number of z-slices in the volume segment) */
-  tiles: number;
-  /** Physical x size of a single *original* (not downsampled) pixel */
-  pixel_size_x: number;
-  /** Physical y size of a single *original* (not downsampled) pixel */
-  pixel_size_y: number;
-  /** Physical z size of a single pixel */
-  pixel_size_z: number;
-  /** Symbol of physical unit used by `pixel_size_(x|y|z)` fields */
-  pixel_size_unit: string;
-
-  // backwards compatibility: these are optional
-  /** Full x size of the volume. `tile_width` will be used if unspecified */
-  vol_size_x?: number;
-  /** Full y size of the volume. `tile_height` will be used if unspecified */
-  vol_size_y?: number;
-  /** Full z size of the volume. `tiles` will be used if unspecified */
-  vol_size_z?: number;
-  /** x offset of the volume subset into the volume */
-  offset_x?: number;
-  /** y offset of the volume subset into the volume */
-  offset_y?: number;
-  /** z offset of the volume subset into the volume */
-  offset_z?: number;
-
-  channels: number;
-  channel_names: string[];
-  channel_colors?: [number, number, number][];
+  version?: string;
+
+  /** XY size of the *original* (not downsampled) volume, in pixels */
+  // If we ever allow downsampling in z, replace with Vector3
+  originalSize: Vector2;
+  /** Number of rows and columns of z-slice tiles (not pixels) in the texture atlas */
+  atlasDims: Vector2;
```

Comment:
Something about this comment doesn't really connect for me. 
I *think* I get it (it's how each Z-slice is laid out in the texture atlas), but I'm wondering if there's a more descriptive way to illustrate what you mean here.

Reply:
> Something about this comment doesn't really connect for me. I _think_ I get it (it's how each Z-slice is laid out in the texture atlas), but I'm wondering if there's a more descriptive way to illustrate what you mean here.

I think the variable name `atlasDims` is hard to understand at a glance.  This is a tricky one to name.  `numTilesXY` or somehow using the word tiles instead of atlas might help.   To me, the comment does describe what the variable is, pretty well.  Maybe it could be
```
// number of tiles(z slices) in x and y of the texture atlas
```